
    try
    {
        console.log('Oi');
    }
    catch(error)
    {
        console.log(error);
    }
    console.log("Funcionou");